mesonbuild/msubprojects.py

```diff
@@ -155,7 +155,7 @@ def update_wrapdb(self) -> bool:
         try:
             wrapdb_version = self.wrap.get('wrapdb_version')
             branch, revision = wrapdb_version.split('-', 1)
-        except WrapException:
+        except (WrapException, ValueError):
```

Comment:
IMO we should not handle ValueError here as equivalent to missing, but raise an error stating that the wrap file is malformed.

Reply:
Changed to catch it separately and print a different message.